Makefile

```diff
@@ -1,24 +0,0 @@
-NPM_EXECUTABLE_HOME := node_modules/.bin
```

Comment:
Since this file is gone, `script/cibuild` can go too. It's using this makefile, so it doesn't work now anyway.